code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -752,6 +793,7 @@
 /datum/tarot/reversed/the_hermit
 	name = "IX - The Hermit?"
 	desc = "May you see the value of all things in life."
+	extended_desc = "will sell all loose guns, grenadeds, battons, and armour around the user, directly into cash."
```

Comment:
```suggestion
	extended_desc = "will sell all loose guns, grenades, batons, and armor around the user, transforming them directly into cash."
```
I think we use American English for the game itself and European English for the wiki? it's confusing, I know.